examples/dataflow-flex-python/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# This Dockerfile is for creating an Image installed with all
+# dependencies that will be used by Dataflow workers at run time.
+
+FROM python:3.8-slim
+
+RUN pip install --no-cache-dir apache-beam[gcp]==2.40.0 xmltodict==0.13.0
+
+RUN pip check
+
+COPY --from=apache/beam_python3.8_sdk:2.40.0 /opt/apache/beam /opt/apache/beam
+
+ENTRYPOINT ["/opt/apache/beam/boot"]
```

Comment:
Simplify this by using apache/beam_python3.8_sdk:2.40.0 as the base image and just adding a RUN ... line.
You'd want to reuse the existing requirements.txt to avoid having to update dependencies all over the place in both Dockerfiles and requirements.txt.

Reply:
Using apache/beam_python3.8_sdk:2.40.0 now and installed dependencies on top of that.